wrapper/src/main/java/software/amazon/jdbc/Driver.java

```diff
@@ -96,10 +96,21 @@ public Connection connect(final String url, final Properties info) throws SQLExc
     LOGGER.finest("Opening connection to " + url);
 
     final String driverUrl = url.replaceFirst(PROTOCOL_PREFIX, "jdbc:");
-    final java.sql.Driver driver = DriverManager.getDriver(driverUrl);
+
+    final String message = Messages.get(
+        "Driver.missingDriver",
+        new Object[] {driverUrl, Collections.list(DriverManager.getDrivers())});
+
```

Comment:
Nit: Would be nice to move the creation of the error message into a method. Then calling the method when its needed. That way dont need to create the error message when it might not be needed.